apps/web/screens/Applications/Applications.tsx

```diff
@@ -0,0 +1,331 @@
+                        query: { ...query, page },
+                      }}
+                    >
+                      <span className={className}>{children}</span>
+                    </Link>
+                  )}
+                />
+              </Box>
+            )}
+          </GridColumn>
+        </GridRow>
+      </GridContainer>
+    </>
+  )
+}
+
+Applications.getProps = async ({ apolloClient, locale, query }) => {
+  // TODO - file bugreport with next-usequerystate regarding missing default on server
+  const {
+    q = '*',
+    page: pageParam = '1',
+    category: categoryParam,
+    organization: organizationParam,
+  } = query
+  const queryString = parseAsString.withDefault('*').parseServerSide(q)
+  const page = parseAsInteger.withDefault(1).parseServerSide(pageParam)
+  const order = SortDirection.Desc
+  const category = parseAsArrayOf(parseAsString)
+    .withDefault([])
+    .parseServerSide(categoryParam)
+  const organization = parseAsArrayOf(parseAsString)
+    .withDefault([])
+    .parseServerSide(organizationParam)
+  const types = [SearchableContentTypes['WebArticle']]
+  const tags: TagType[] = [
+    ...category.map(
+      (key: string): TagType => ({
+        type: SearchableTags.Category,
+        key,
+      }),
+    ),
+    ...organization.map(
+      (key: string): TagType => ({
+        type: SearchableTags.Organization,
+        key,
+      }),
+    ),
+    {
+      type: SearchableTags.Processentry,
+      key: 'true',
+    },
+  ]
+
+  const [
+    {
+      data: { searchResults },
+    },
+    {
+      data: { searchResults: countResults },
+    },
+    namespace,
+  ] = await Promise.all([
+    apolloClient.query<GetApplicationsQuery, QuerySearchResultsArgs>({
+      fetchPolicy: 'no-cache', // overriding because at least local caching is broken
+      query: GET_APPLICATIONS_QUERY,
+      variables: {
+        query: {
+          language: locale as ContentLanguage,
+          queryString,
+          sort: SortField.Popular,
+          order,
+          types,
+          ...(tags.length && { tags }),
+          size: PERPAGE,
+          page,
+        },
+      },
+    }),
+    apolloClient.query<GetSearchCountTagsQuery, QuerySearchResultsArgs>({
+      fetchPolicy: 'no-cache', // overriding because at least local caching is broken
+      query: GET_SEARCH_COUNT_QUERY,
+      variables: {
+        query: {
+          language: locale as ContentLanguage,
+          queryString,
+          countTag: [SearchableTags.Category, SearchableTags.Organization],
+          types,
+          countProcessEntry: true,
+        },
+      },
+    }),
+    apolloClient
+      .query<GetNamespaceQuery, QueryGetNamespaceArgs>({
+        query: GET_NAMESPACE_QUERY,
+        variables: {
+          input: {
+            namespace: 'Applications',
+            lang: locale,
+          },
+        },
+      })
+      .then((variables) =>
+        JSON.parse(variables?.data?.getNamespace?.fields || '[]'),
```

Comment:
Why not default to a '{}' instead?

Reply:
Same as above. Copy-pasta issue. Will change. I want to also add type safety throughout the file where it's missing. Linter should have picked this up since it's defaulting to `any`.